docs/kilt-baseline-replication.md

```diff
@@ -0,0 +1,136 @@
+# Replication guide KILT
+
+## Setup env
+
+```bash
+# Create a virtual env
+conda create -n kilt37 -y python=3.7 && conda activate kilt37
```

Comment:
We're using Python 3.6 project wide, can you please switch?

Reply:
This is the environment KILT uses, so I'm not sure if there'd be any dependency issues with their scripts by switching to 3.6. My current setup uses both scripts in pyserini and scripts from the KILT repository.

Reply:
okay, let's keep then.